### PR TITLE
Add space between tweet and link

### PR DESCRIPTION
**Pull request by georgeglessner**
Adding on to @aldnav's pull request, add a space between tweet and link.

### CODE REVIEW

File: twitter_scraper.py [hunk 1]
```diff
@@ -16,15 +16,18 @@ def gen_tweets(pages):
...
             for tweet in tweets:
+                if 'http' in tweet:
+                    tweet = tweet.split('http',1)
+                    tweet = tweet[0] + ' http' + tweet[1]
                 if tweet:
                     yield tweet
 
-            r = requests.get(url, params={'max_position': last_tweet}, headers=headers)
+            r = requests.get(
+            url, params={'max_position': last_tweet}, headers=headers)
```
Comment (aldnav) [inline twitter_scraper.py]: I think its better that this line stays indented

Comment (georgeglessner) [reply twitter_scraper.py]: Yeah, sorry didn't mean for that to happen!

Comment (aldnav) [reply twitter_scraper.py]: No worries :+1:

File: twitter_scraper.py [hunk 2]
```diff
@@ -16,15 +16,18 @@ def gen_tweets(pages):
 
         while pages > 0:
             d = pq(r.json()['items_html'])
-
-            tweets = [tweet.text for tweet in d('.tweet-text')]
+            tweets = [tweet.text_content() for tweet in d('.tweet-text')]
             last_tweet = d('.stream-item')[-1].attrib['data-item-id']
 
             for tweet in tweets:
+                if 'http' in tweet:
```
Comment (aldnav) [inline twitter_scraper.py]: For [lines 23-25](https://github.com/georgeglessner/twitter-scraper/blob/4ce16ce14c495a55b1cf2151400ecf277b2d4770/twitter_scraper.py#L23-L25), how about this?

```py
# 'check this outhttp://from-http-to-https'
tweet = re.sub('http', ' http', tweet, 1)
# 'check this out http://from-http-to-https'
```

Just throwing suggestions.

**Comment by aldnav** [comment]
@georgeglessner Also, I don't know what this is https://github.com/georgeglessner/twitter-scraper/blob/4ce16ce14c495a55b1cf2151400ecf277b2d4770/__pycache__/twitter_scraper.cpython-36.pyc , perhaps you can also add `.gitignore` for python project (maybe on a separate branch too)

**Comment by georgeglessner** [comment]
Added changes to a new branch called enhancements

**Pull request closed without merging**